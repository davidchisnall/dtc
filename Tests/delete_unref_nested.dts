/dts-v1/;

/ {

	/omit-if-no-ref/ out: outer@2 {
		inner@4 {
			inner@1 {
				test-property = <&out>;
			};
		};
	};
};
