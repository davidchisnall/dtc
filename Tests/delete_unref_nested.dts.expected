/dts-v1/;

/  {

	outer@2 {

		phandle = <0x1>;
		inner@4 {

			inner@1 {

				test-property = <0x1>;
			};
		};
	};
};
